code/weapon/weapon.h

```diff
@@ -655,8 +664,8 @@ void shield_impact_explosion(vec3d *hitpos, object *objp, float radius, int idx)
 // Swifty - return number of max simultaneous locks 
 int weapon_get_max_missile_seekers(weapon_info *wip);
 
-// return if this weapon can lock on this ship type
-bool weapon_can_lock_on_ship_type(weapon_info *wip, int ship_type);
+// return if this weapon can lock on this ship, based on its type, class or species
+bool weapon_multilock_can_lock_on_ship(weapon_info *wip, int ship_num);
```

Comment:
The `ship_num` parameter should be renamed as it's a multi-purpose parameter now.

Reply:
I disagree considering `ship_num` is literally a index into `Ships`

Reply:
I'm sorry, you're right.  I mentally mixed up two parts of the API.